core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1179,12 +1179,12 @@ public CommitId flush(boolean force, boolean waitIfOngoing) throws EngineExcepti
                     try {
                         translog.prepareCommit();
                         logger.trace("starting commit for flush; commitTranslog=true");
-                        commitIndexWriter(indexWriter, translog, null);
+                        final long committedGeneration = commitIndexWriter(indexWriter, translog, null);
```

Comment:
I think all of them suffer from the same problem. This is just the first one I run into while reviewing.  I think I get why it's named like this now (the translog id that was committed into lucene's metadata) but there is much room for ambiguity - is the the maximum generation for which all ops are in lucene (i.e., committed)?  I wonder every time and end up double checking the code. If people find this name better, I'm good with keeping it - it's a subjective matter.